CommonTools/ParticleFlow/python/pfNoPileUpJME_cff.py

```diff
@@ -18,3 +19,8 @@
     pfNoPileUpJME
     )
 
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(
+    pfPileUpJME,
+    Vertices = cms.InputTag("offlinePrimaryVertices"),
```

Comment:
I can take a closer look, but the goodPrimaryVertices filtering was not working properly with the 4d vertices.  (Giving 0 output vertices.)